test_run.sh

```diff
@@ -112,6 +112,16 @@ xmllint_test() {
   find -L ./bazel-testlogs -iname "*.xml" | xargs -n1 xmllint > /dev/null
 }
 
+multiple_junit_suffixes() {
+  bazel test //test:JunitMultipleSuffixes
+  matches=$(grep -c -e 'Running E2E' -e 'Running IT' ./bazel-testlogs/test/JunitMultipleSuffixes/test.log)
+  if [ $matches -eq 2 ]; then
+    return 0
+  else
+    return 1
+  fi
+}
```

Comment:
I just mean that bazel expects test xml on a certain path:
https://github.com/bazelbuild/rules_scala/blob/master/scala/support/JUnitXmlReporter.scala#L82

so, I wonder if we can see that we are generating it on the right path.